data/modules/DebugShipSpawn/DebugShipSpawn.lua

```diff
@@ -0,0 +1,107 @@
+local Game = require('Game')
+local Space = require('Space')
+local Ship = require('Ship')
+local ShipDef = require('ShipDef')
+local Color = require('Color')
+local Equipment = require('Equipment')
+local Vector2
+Vector2 = _G.Vector2
+local ui = require('pigui.pigui')
+local ship_defs = { }
+local update_ship_def_table
+update_ship_def_table = function()
+  for name in pairs(ShipDef) do
+    table.insert(ship_defs, name)
+    print("Ship Def found: " .. tostring(name))
+  end
+  return table.sort(ship_defs)
+end
+update_ship_def_table()
+print(#ship_defs)
+local selected_ship_type = 0
+local draw_ship_types
+draw_ship_types = function()
+  for i, ship in ipairs(ship_defs) do
+    if ui.selectable(ship, selected_ship_type == i) then
```

Comment:
After the refactoring of flag handling this line causes a crash. Flags either need to be explicitly passed here, or `l_pigui_selectable` in `LuaPiGui.cpp` should be changed to use `LuaPull` with a default value.

Reply:
Hmm. I thought I changed `l_pigui_selectable` to use the default value. Must not have survived the rebase.

Regarding indentation, that's because this file is autogenerated from `DebugShipSpawner.moon` in the same directory - I'll see what I can do about that.